app/controllers/paypal_agreement_controller.rb

```diff
@@ -0,0 +1,32 @@
+class PaypalAgreementController < ApplicationController
+  def new
+    @plan = Plan.find(params[:plan])
+    binding.pry
```

Comment:
@mattwr18 was just trying to check travis for the latest error, but saw you have  a binding.pry in here - part of the latest round of debugging.  I'll remove locally and check, but I was just wondering if the latest code you and Ryder were looking at is in the branch, or do you have some additional change locally that I should have before I start my review?

Reply:
that's the latest code @tansaku, I thought I had removed all the `binding.pry`s, but they are sneaky sometimes.

there are some things that I know need to be changed, like the `redirect_url` from here https://github.com/AgileVentures/WebsiteOne/pull/3018/files#diff-8ba20331054f486b79d6002c4ebd45a1R6 does not match the one from the fixtures here https://github.com/AgileVentures/WebsiteOne/pull/3018/files#diff-bb17bf70169813026eb3e62c87ef480dR40, but that is not what I am blocked by